WORKSPACE

```diff
@@ -181,8 +181,8 @@ git_repository(
 )
 
 bind(
```

Comment:
I don't remember, can we do list comprehension in WORKSPACE? Lets dedup if we can?

Reply:
Turns out we can. Updated this, at least the three `absl/container:*` dependencies are deduplicated now.